music_assistant/helpers/tags.py

```diff
@@ -525,6 +525,9 @@ def parse_tags(
             extra_tags = parse_tags_mutagen(input_file)
             if extra_tags:
                 tags.tags.update(extra_tags)
+            # APEv2 cover art is not exposed as video streams by FFmpeg, check via mutagen
+            if not tags.has_cover_image and has_apev2_cover_art(input_file):
```

Comment:
can we avoid doing this call or make it return early if we know the tags are not APEv2 ?
this will slow down the sync big time when there are a lot of tracks without embedded image

Reply:
Logic added to only do this check on files which exclusively use APEv2 tags